docs/gen-client-docs.py

```diff
@@ -54,6 +54,23 @@ def __str__(self):
         return ret
 
 
+@dataclass
+class PropertyDoc:
+    name: str
+    doc: Optional[str]
+    tpe: Optional[str]
+
+    def as_rst(self) -> str:
+        out = ['', f'    .. py:property:: {self.name}']
+        if self.tpe is not None:
+            out.append(f'        :type: {self.tpe}')
+        if self.doc is not None:
+            # This is a class doc, which comes with 4 indentation spaces. 
+            # Here we are using the doc as property doc, which needs 8 indentation spaces.
+            formatted_doc = re.sub(r'\n', '\n    ', self.doc)
+            out.append(f'\n        {formatted_doc}')
```

Comment:
I added `_get_enum_doc` recently that basically does this. Let's see if we can refactor/reuse that method?

Reply:
That method also strips double returns to keep everything as a single paragraph.